### PR TITLE
Fixed PersistentVolumeClaim spelling error

### LINKED ISSUE #1993: PersistentVolumeClaim Spelling Error
The PersistentVolumeClaim attribute of a function's volume name is mis-spelled `presistent`.

https://github.com/knative/func/blob/main/pkg/functions/function_volumes.go#L8

### PR DESCRIPTION
**Pull request by prushh**
Fixes #1993

### CODE REVIEW

File: schema/func_yaml-schema.json [hunk 1]
```diff
@@ -1,420 +0,0 @@
-{
```
Comment (matejvasek) [inline schema/func_yaml-schema.json:1]: Why is this file deleted?

Comment (prushh) [reply schema/func_yaml-schema.json]: Sorry, I accidentally deleted that file...

I hope everything is okay now!

Comment (prushh) [reply schema/func_yaml-schema.json]: One clarification: I built the project following the [CONTRIBUTING.md](https://github.com/knative/func/blob/main/docs/CONTRIBUTING.md) guide.
In particular, I ran `make build` -> `make clean` -> `make test`.

Is it correct that the `make clean` command deletes the _schema/func_yaml-schema.json_ file?
https://github.com/knative/func/blob/5bb373ab2ca4169437daf011f087b42c47960480/Makefile#L104C1-L109C20

Comment (matejvasek) [reply schema/func_yaml-schema.json]: I am not sure why it's deleting that file.

**Comment by matejvasek** [comment]
PTAL @jrangelramos @lkingland @gauron99

**Comment by matejvasek** [comment]
I added migration function https://github.com/knative/func/pull/2069/commits/682f56876007cb74167ea7dad058f37ca6af46e6.

**Comment by prushh** [comment]
Thanks @matejvasek!
